metaflow/__init__.py

```diff
@@ -219,43 +219,43 @@ def _can_handle_orig_module():
 # Now override everything other than modules
 __version_addl__ = None
 try:
-    import metaflow_custom.toplevel.toplevel as extension_module
+    import metaflow_extensions.toplevel.toplevel as extension_module
 except ImportError as e:
     ver = sys.version_info[0] * 10 + sys.version_info[1]
     if ver >= 36:
         # e.name is set to the name of the package that fails to load
         # so don't error ONLY IF the error is importing this module (but do
         # error if there is a transitive import error)
         if not (isinstance(e, ModuleNotFoundError) and \
-                e.name in ['metaflow_custom', 'metaflow_custom.toplevel',
-                           'metaflow_custom.toplevel.toplevel']):
+                e.name in ['metaflow_extensions', 'metaflow_extensions.toplevel',
```

Comment:
@romain-intel This change may require a corresponding change in the nflx-internal metaflow-extensions.

Reply:
There is one missing there (the toplevel.toplevel). OK for nflx package.

Reply:
oh nvm, it was below.